resources/hiveclients/python/hiveserver1-client.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c